src/distraction/DroppingBox.java

```diff
@@ -0,0 +1,104 @@
+package distraction;
+
+import com.jme3.asset.AssetManager;
+import com.jme3.bullet.BulletAppState;
+import com.jme3.bullet.control.RigidBodyControl;
+import com.jme3.material.Material;
+import com.jme3.math.ColorRGBA;
+import com.jme3.math.Vector3f;
+import com.jme3.scene.Geometry;
+import com.jme3.scene.Spatial;
+import com.jme3.scene.shape.Box;
+import eu.opends.car.SteeringCar;
+import eu.opends.main.Simulator;
+/**
+ *
+ * @author Johnny Marek
+ */
+public class DroppingBox extends DistractionClass{
+    
+    private Simulator sim;
+    private SteeringCar car;
+    private AssetManager manager;
+    private Geometry DropBox;
+    private Spatial dropingBox;
+    private BulletAppState bulletAppState;
+    private RigidBodyControl box_phy;
+    private float time;
+    private float m = 0;
+    private Vector3f carPos;
+    
+    
+    public DroppingBox(){
+        
+}
+    
+    @Override
+    public void initialize(Simulator sim) {
+        this.sim = sim;
+        this.car = sim.getCar();
+        this.manager = sim.getAssetManager();
+        this.bulletAppState = sim.getBulletAppState();
+        
+        box_phy = new RigidBodyControl(2f);
+           
+        Box box = new Box(1, 1, 1f);  
```

Comment:
make the relevant values (eg. the sizes here(?)) parametric, passed to constructor.